src/it/monitor-with-default-phase/verify.groovy

```diff
@@ -2,6 +2,14 @@ import org.codehaus.plexus.util.FileUtils;
 
 String log = FileUtils.fileRead(new File(basedir, "build.log"));
 
+if (!log.contains("snyk executable path: " + System.getenv("SNYK_CLI_EXECUTABLE"))) {
```

Comment:
These assertions fail on Windows likely because of carriage return not being captured. It might be more clear to log the entire thing in one line anyway like "Snyk CLI Version x.x.x".